node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/ScalingSuggestionsMaintainer.java

```diff
@@ -36,13 +36,16 @@ public ScalingSuggestionsMaintainer(NodeRepository nodeRepository,
     }
 
     @Override
-    protected boolean maintain() {
-        if ( ! nodeRepository().zone().environment().isProduction()) return true;
+    protected double maintain() {
+        if ( ! nodeRepository().zone().environment().isProduction()) return 1.0;
 
+        int attempts = 0;
         int successes = 0;
-        for (var application : activeNodesByApplication().entrySet())
+        for (var application : activeNodesByApplication().entrySet()) {
+            attempts++;
             successes += suggest(application.getKey(), application.getValue());
-        return successes > 0;
+        }
+        return attempts == 0 ? 1.0 : ((double)successes / attempts);
```

Comment:
Use `asSuccessFactor` here too?

Reply:
Here is was easier to count successes than failures.